tools/run_tests/run_interop_tests.py

```diff
@@ -1230,6 +1230,11 @@ def aggregate_http2_results(stdout):
       _HTTP2_TEST_CASES, http2_server_test_cases, resultset, num_failures,
       args.cloud_to_prod_auth or args.cloud_to_prod, args.prod_servers,
       args.http2_interop)
+  
+  if num_failures:
```

Comment:
I did a test and it proves to work: https://kokoro.corp.google.com/job/grpc/job/ubuntu/job/pull_request/job/grpc_interop_toprod/25/